setup.py

```diff
@@ -68,6 +68,7 @@
     f'invenio[base,auth,metadata,files]{invenio_version}',
     'invenio-logging[sentry-sdk]>=1.3.0,<1.4.0',
     'invenio-rdm-records>=0.35.1,<0.36.0',
+    'invenio-search-ui>=1.2.0,<1.3.0',
```

Comment:
it is already part of `invenio` above, so it should not be needed. Can you please check?

Reply:
You are right not needed anymore as it is part of `invenio`